content/soups and stews/Middle-Eastern Chili.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Middle-Eastern Chili
+category: soups and stews
+date: 2025-08-21
+recipeYear: "2025"
+recipeDecade: 2020s
+author: Liza Desyatkova
+prepTime: 15m
+cookTime: 30m
+vegetarian: true
+---
+
+### Ingredients
+
+#### Part 1
+
+- 1 16 ounce can chickpeas, rinsed and drained
+- 1 16 ounce can fava beans beans, rinsed and drained
+- 1 28 ounce can crushed tomatoes
+- 1 12 ounce can of corn
+- 2 1/2 cups soup broth of choice
+- 1 small red bell pepper diced
+- 1 medium yellow onion diced
+- 1 small lemon, juiced
+- 4 cloves minced garlic
+- 2 tablespoons of olive oil
+- 2 tablespoons chili powder
+- 1 tablespoon curry powder 
+- 1 tablespoon paprika
+- 3 teaspoon dried cumin
+- 2 teaspoons ground coriander
+- 2 teaspoons of ground masala
+- 1 teaspoon ground cardamom
+- 1 teaspoon ground cloves
+- 1 teaspoon ground allspice
+- 2 bay leafs
+- Bread of choice
+- **Optional:** BBQ or Dijon sauce for extra flavour
+
+---
+
+### Instructions
+
+1. Prepare large pot with olive oil on medium heat 
```

Comment:
Please remove trailing spaces at the end of lists.  Instructional lists should also have periods at the end.

```suggestion
1. Prepare large pot with olive oil over medium heat.
```